apps/docs/content/php/how-to/customize-runtime.mdx

```diff
@@ -50,3 +50,15 @@ To invalidate the custom runtime cache go to `yyy`
 **// TODO screenshot**
 
 When the custom runtime cache is used, Zerops doesn't create a prepare runtime container and executes the deployment of your application directly.
+
```

Comment:
I would add the description of what example it is of. 

Add a heading e.g. ### Overwrite php.ini files and add this :point_down: it looks fine, just clean it up a little so it sounds good (we don't use "I" in the docs etc)
It is possible to overwrite php.ini values using env variables. If I want to overwrite the `post_max_size` variable to a value of 10M, I can do this by creating a `PHP_INI_post_max_size` variable with a value of 10M. In zerops.yaml for example: